src/main/java/dev/blaauwendraad/masker/json/JsonMasker.java

```diff
@@ -17,7 +16,6 @@ public interface JsonMasker {
      * @param targetKeys the key(s) to target
      * @return the {@link JsonMasker} instance
      */
-    @Nonnull
```

Comment:
@gavlyukovskiy I just thought a bit more about it. I this we should now, at least to the main methods that form the API of the library, but in the `@return` JavaDoc that it will not return null.

Before developers could see this from the ` @Nonnull` annotations. Now they need to know that we use NullAway (which most users probably won't actually look at)

Reply:
I honestly think we would be overdoing it and making the docs harder to read. Even if the developer doesn't use an IDE, that would scream about redundant null-check, it should be obvious that a factory method `getMasker` doesn't return `null`, adding that to the doc is quite useless in my opinion.